internal/command/meta_backend_migrate.go

```diff
@@ -1003,21 +1004,22 @@ For example, if a workspace is currently named 'prod', the pattern 'app-*' would
 
 const tfcInputBackendMigrateMultiToMulti = `
 Unlike typical Terraform workspaces representing an environment associated with a particular
-configuration (e.g. production, staging, development), Terraform Cloud workspaces are named uniquely
-across all configurations used within an organization. A typical strategy to start with is
-<COMPONENT>-<ENVIRONMENT>-<REGION> (e.g. networking-prod-us-east, networking-staging-us-east).
+configuration (e.g. production, staging, development), HCP Terraform and Terraform Enterprise 
+workspaces are named uniquely across all configurations used within an organization. A typical 
+strategy to start with is <COMPONENT>-<ENVIRONMENT>-<REGION> (e.g. networking-prod-us-east, 
+networking-staging-us-east).
 
 For more information on workspace naming, see https://www.terraform.io/docs/cloud/workspaces/naming.html
 
-When migrating existing workspaces from the backend %[1]q to Terraform Cloud, would you like to
-rename your workspaces? Enter 1 or 2.
+When migrating existing workspaces from the backend %[1]q to HCP Terraform or Terraform Enterprise, 
+would you like to rename your workspaces? Enter 1 or 2.
```

Comment:
Hmm, this instance of the "x or y" compromise is somewhat more frustrating than most of the others. 

We don't currently have the name available in the `promptMultiStateMigrationPattern` helper function, but that func's call site _does_ have access to a full cloud backend (in opts.Destination), so it ought to be possible to do a type-assert, grab the .AppName() off it, and pass it into the helper as a second arg. 

Reckon it's worth it? The approach would also be applicable to some of the other messages in this zone.